colibre/registration.py

```diff
@@ -143,9 +145,46 @@
 total_dust_mass = total_dust_fraction * catalogue.masses.m_gas
 name = f"$M_{{\\rm dust}}${dust_frac_error}"
 total_dust_mass.name = name
-nonmetal_frac = 1. - catalogue.apertures.zmet_gas_sf_100_kpc
+total_dust_fraction.name = f"$\\mathcal{{DTG}}${dust_frac_error}"
+total_dust_mass = total_dust_fraction * catalogue.masses.m_gas
```

Comment:
Should we be using the gas mass in the 100kpc aperture here?

Reply:
use gas mass now